src/Entities/Payout/ExternalAccountBeneficiary.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TrueLayer\Entities\Payout;
+
+use TrueLayer\Constants\BeneficiaryTypes;
+use TrueLayer\Entities\Entity;
+use TrueLayer\Interfaces\AccountIdentifier\AccountIdentifierInterface;
+use TrueLayer\Interfaces\AddressInterface;
+use TrueLayer\Interfaces\Payout\ExternalAccountBeneficiaryInterface;
+
+final class ExternalAccountBeneficiary extends Entity implements ExternalAccountBeneficiaryInterface
+{
+    /**
+     * @var string
+     */
+    protected string $accountHolderName;
+
+    /**
+     * @var AccountIdentifierInterface
+     */
+    protected AccountIdentifierInterface $accountIdentifier;
+
+    /**
+     * @var string
+     */
+    protected string $reference;
+
+    /**
+     * @var string
+     */
+    protected string $dateOfBirth;
+
+    /**
+     * @var AddressInterface
+     */
+    protected AddressInterface $address;
+
+    /**
+     * @var string[]
+     */
+    protected array $casts = [
+        'account_identifier' => AccountIdentifierInterface::class,
+        'address' => AddressInterface::class,
+    ];
+
+    /**
+     * @var string[]
+     */
+    protected array $arrayFields = [
+        'account_holder_name',
+        'account_identifier',
+        'reference',
+        'type',
+        'date_of_birth',
+        'address',
+    ];
+
+    /**
+     * @return string|null
+     */
+    public function getAccountHolderName(): ?string
```

Comment:
Can we match these methods to the api spec so that for fields marked as `required` the return type isn't nullable? Fo example reference, acc holder name are required.